### PR TITLE
Feature/quiz template pagiantion

### LINKED ISSUE #7: 006-feature-pagination-quizzes
Activates pagination in quizzes

### PR DESCRIPTION
**Pull request by ApostolaT**
Pagination for quizzes on admin dashboard and on user quizzes listing

### CODE REVIEW

File: src/Controllers/LoginController.php [hunk 1]
```diff
@@ -41,7 +41,7 @@ public function loginAction(RequestInterface $request)
             if ($this->session->get('role') === 'user') {
                 $response = new \Framework\Http\Response(Stream::createFromString(' '), []);
                 $response = $response->withStatus(301);
-                $response = $response->withHeader('Location', 'http://local.quiz.com/quiz/1');
+                $response = $response->withHeader('Location', 'http://local.quiz.com/quiz');
 
                 return $response;
```
Comment (ApostolaT) [inline src/Controllers/LoginController.php:46]: I will use the getRedirectPage from abstract controller, now it's not the goal of the current branch

File: src/Controllers/QuizController.php [hunk 2]
```diff
@@ -46,20 +46,39 @@ public function setQuestionService(AbstractService $questionService)
         $this->questionService = $questionService;
     }
 
+    /**
+     * Gets all the entities and displays them on them either on
+     * admin quizzes page or on user quizzes page
```
Comment (ApostolaT) [inline src/Controllers/QuizController.php:51]: I will fix this on the next commit

File: src/Controllers/QuizController.php [hunk 3]
```diff
@@ -98,14 +124,20 @@ public function addQuiz(RequestInterface $request)
         return $this->renderer->renderView("login.phtml", $request->getRequestParameters());
```
Comment (ApostolaT) [inline src/Controllers/QuizController.php:124]: I will use a redirect here when i will start the refactoring issue. Now it's not the goal of the pull request

File: src/Controllers/LoginController.php (hunk 1 above)
Comment (victortodoran) [reply src/Controllers/LoginController.php]: ok

File: src/Controllers/QuizController.php (hunk 2 above)
Comment (victortodoran) [reply src/Controllers/QuizController.php:51]: This comment refers to an issue with the language used in the comments. Ok.

File: src/Controllers/QuizController.php (hunk 3 above)
Comment (victortodoran) [reply src/Controllers/QuizController.php:124]: ok

**Pull request merged**